doc/rst/language/spec/data-parallelism.rst

```diff
@@ -278,12 +278,14 @@ variable, the resulting array has a 1-based one-dimensional domain.
 
       ;
       writeln(result);
+      writeln(result.domain);
```

Comment:
I thought for sure this would be a `.indices` call :)

Reply:
Gotta overcome decades of bad practices... :)